libravatar/libravatar.php

```diff
@@ -81,26 +70,15 @@ function libravatar_addon_admin(&$a, &$o)
 		'monsterid' => DI::l10n()->t('monster face'),
 		'wavatar' => DI::l10n()->t('computer generated face'),
 		'retro' => DI::l10n()->t('retro arcade style face'),
+		'robohash' => DI::l10n()->t('roboter face'),
```

Comment:
You should regenerate the addons `messages.po` file after these adds.

Reply:
What are the correct parameters for the addons?

Reply:
see [here](https://github.com/friendica/friendica/blob/2020.12-rc/doc/translations.md#addon) for the two variants. Either `-s` in the addons directory or `-a addon_name`.

Reply:
Done